buildspec.yml

```diff
@@ -2,16 +2,16 @@ version: 0.2
 
 env:
   variables:
-    FRAMEWORK_VERSION: '1.10.2'
+    FRAMEWORK_VERSION: '2.0.1'
```

Comment:
Is this the DLC version? https://github.com/aws/deep-learning-containers/blob/master/pytorch/inference/buildspec.yml#L4

Reply:
This version corresponds to the PyTorch version in the DLC images. For our testing, we maintain two DLC images (CPU and GPU) for the particular version. For example, for `2.0.1`, we pull the following images and re-install the PyTorch toolkit binary to get the custom binary:
1. `763104351884.dkr.ecr.$region.amazonaws.com/pytorch-inference:2.0.1-cpu-py310-ubuntu20.04-sagemaker`
2.  `763104351884.dkr.ecr.$region.amazonaws.com/pytorch-inference:2.0.1-gpu-py310-cu118-ubuntu20.04-sagemaker`